apps/ejabberd/src/mod_muc_admin.erl

```diff
@@ -101,7 +116,8 @@ create_instant_room(Host, Name, Owner, Nick) ->
     %% Send presence to create a room.
     ejabberd_router:route(OwnerJID, UserRoomJID, presence()),
     %% Send IQ set to unlock the room.
-    ejabberd_router:route(OwnerJID, BareRoomJID, declination()),
```

Comment:
I know that this question is a bit out of scope of this PR but could someone remind me why do we send presence in order to create a room instead of calling `mod_muc:create_instant_room/5` ?